src/index.html

```diff
@@ -4,10 +4,58 @@
     <meta charset="UTF-8">
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <title>Product cards</title>
+    <title>Stars</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+
+    <div class="stars--0">
+      <div class="star__primary"></div>
```

Comment:
again wrong BEM
https://en.bem.info/methodology/quick-start/
read about blocks and elements
you can't use element name with `star` prefix since you don't have a parent block class `star`

Reply:
fixed